src/main/java/com/invicta/impl/hack/hacks/player/SprintHack.java

```diff
@@ -0,0 +1,25 @@
+package com.invicta.impl.hack.hacks.player;
+
+import com.invicta.api.hack.AbstractHack;
+import com.invicta.api.hack.HackCategory;
+
+/**
+ * @author Darki 25/Dec/2019
+ */
+
+@AbstractHack.Hack(name = "Sprint", description = "Makes you sprint!", category = HackCategory.COMBAT)
+public class SprintHack extends AbstractHack {
+	
+	
+	@Override
+	public void onUpdate() {
+		if(this.isEnabled()) {
+			Minecraft.getMinecraft().player.setSprinting() = true;
+		}
+	}
+	
+	@Override
+	public void onDisable() {
+		Minecraft.getMinecraft().player.setSprinting() = false;
```

Comment:
If the player is using the sprint hack module and disables it while holding their sprint key it will stop them sprinting even if their sprint key is down. You should instead set it to true/false based on whether they are pressing the sprint key.

Reply:
Ok will do 😳